service/frontend/operator_handler.go

```diff
@@ -276,6 +282,61 @@ func (h *OperatorHandlerImpl) ListSearchAttributes(ctx context.Context, request
 	}, nil
 }
 
+func (h *OperatorHandlerImpl) DeleteNamespace(ctx context.Context, request *operatorservice.DeleteNamespaceRequest) (_ *operatorservice.DeleteNamespaceResponse, retError error) {
+	const endpointName = "DeleteNamespace"
+
+	defer log.CapturePanic(h.logger, &retError)
+
+	scope, sw := h.startRequestProfile(metrics.OperatorDeleteNamespaceScope)
+	defer sw.Stop()
+
+	// validate request
+	if request == nil {
+		return nil, h.error(errRequestNotSet, scope, endpointName)
+	}
+
+	if request.GetNamespace() == "" {
+		return nil, h.error(errNamespaceNotSet, scope, endpointName)
+	}
+
+	// Execute workflow.
+	wfParams := deletenamespace.DeleteNamespaceWorkflowParams{
+		Namespace: namespace.Name(request.GetNamespace()),
+		DeleteExecutionsConfig: deleteexecutions.DeleteExecutionsConfig{
+			DeleteActivityRPS:                    h.config.DeleteNamespaceDeleteActivityRPS(),
+			ConcurrentDeleteExecutionsActivities: h.config.DeleteNamespaceConcurrentDeleteExecutionsActivities(),
```

Comment:
Workflow has 4 parameters. I decided to expose these two through dynamic config and other 2 just leave workflow defaults.

Reply:
why use dynamic config for this? It would be much easier to just pass in as input.